ebms-http/src/main/kotlin/no/nav/emottak/ebms/App.kt

```diff
@@ -25,9 +25,9 @@ fun main() {
     if (getEnvVar("NAIS_CLUSTER_NAME", "local") != "prod-fss") {
         DecoroutinatorRuntime.load()
     }
-    embeddedServer(Netty, port = 8080, module = Application::ebmsProviderModule, configure = {
-        this.maxChunkSize = 100000
-    }).start(wait = true)
+    embeddedServer(Netty, port = 8080, module = Application::ebmsProviderModule)
+        .also { it.engineConfig.maxChunkSize = 100000 }
+        .start(wait = true)
```

Comment:
Jeg ser at den opprinnelige koden er lettere å lese og forstå. Kanskje vi kan vurdere å beholde den for å gjøre koden mer lesbar og vedlikeholdbar? :thought_balloon:

Reply:
Jeg fikk vite fra Olyksndr at det var nødvendig å endre den pga. Ktor 3 :+1: